src/containers/SearchPage/components/form/SearchContentForm.tsx

```diff
@@ -100,28 +103,36 @@ class SearchContentForm extends Component<Props & tType, State> {
     this.getExternalData();
   }
 
```

Comment:
Jeg hadde foretrekket om denne var en funksjonell komponent. Men vil ikke dytte det på denne pren om det er komplisert å få til.

Jeg ville også fjerna componentDidUpdate om den ikke trengs

Reply:
Ja, funksjonen skal bort. Eg lot den berre stå dersom eg fikk gode argument for at måten eg har gjort det på er veldig feil. Det er det som egentlig er spørsmålet her.

Reply:
Den fanger vel bare ikke opp en endring i searchObject på denne måten. Spørsmålet er vel om det er et problem